games-action/moon-buggy/moon-buggy-1.0.51-r2.ebuild

```diff
@@ -35,7 +35,7 @@ src_configure() {
 
 src_install() {
 	default
-	touch "${D}/var/games/${PN}/mbscore"
+	touch "${D}/var/games/${PN}/mbscore" || die
```

Comment:
```suggestion
	touch "${ED}/var/games/${PN}/mbscore" || die
```

Reply:
On that note, add ${EPREFIX} to /var/games in src_configure too, or else this wouldn't work right.